packages/api/src/main/java/com/osahft/api/job/CleanUpJob.java

```diff
@@ -56,7 +56,19 @@ public void cleanUpEverything() throws LocalFileStorageServiceException, MailTra
                 mailTransferRepository.deleteById(mailTransfer.getId());
             }
         }
+    }
 
+    // run once every second day
+    @Scheduled(fixedRate = 2 * 24 * 60 * 60 * 1000)
+    public void cleanUpLockedMailTransfers() throws LocalFileStorageServiceException, MailTransferRepositoryException {
+        log.info("Cleaning up locked MailTransfers from database");
+        Date oneDayAgo = new Date(System.currentTimeMillis() - 24 * 60 * 60 * 1000);
```

Comment:
shouldn't we do this more often than just once a day?

Reply:
its just deleting the locked transfers once every 2 days, there are no files stored just an object in DB. If clean up more often it could be that the user gets a bad request with the mailtransferid because the id does not exist

Reply:
okay, LGTM then :)